include/powerloader/curl.hpp

```diff
@@ -66,7 +91,7 @@ namespace powerloader
         CURLHandle(const Context& ctx, const std::string& url);
         ~CURLHandle();
 
-        CURLHandle& url(const std::string& url);
+        CURLHandle& url(const std::string& url, const proxy_map_type& proxies);
```

Comment:
I believe the proxy map can be changed after calling this? so this should really be a copy.

Reply:
ok it's only passed to be read